.gitignore

```diff
@@ -23,3 +23,6 @@ go.work.sum
 
 # binaries
 out
+
+contrib/bitcoin
+contrib/regtest
```

Comment:
Let's change to simtest?

Reply:
Actually, we store data in ~/.btcd. :D This .gitignore for bitcoind I wanted to create before.